src/service/jobRoleService.ts

```diff
@@ -12,5 +12,29 @@ export const jobRoleService = {
         catch (e) {
             throw new Error("Failed to get job roles")
         }
+    },
+
+    async getJobRole(id: number): Promise<JobRole> {
+        try {
+            const response = await axios.get(process.env.API_URL + "api/job-roles/" + id);
+            if (response.status === 200) {
+                return response.data
+            }
+        }
+        catch (e) {
+            throw new Error("Failed to get job role")
+        }
+    },
+
+    async deleteJobRole(id: number): Promise<void>{
+        try {
+            const response = await axios.delete(process.env.API_URL + "api/job-roles/" + id);
+            if (response.status === 200) {
+                return response.data
+            }
+        }
+        catch (e) {
+            throw new Error("Failed to delete job role")
+        }
```

Comment:
You may want to log the error messages to the console so you know what the issue is if it doesn't work